src/Aquifer.API/Endpoints/Resources/Content/MachineTranslation/Update/Endpoint.cs

```diff
@@ -19,9 +20,14 @@ public override async Task HandleAsync(Request req, CancellationToken ct)
         var user = await userService.GetUserFromJwtAsync(ct);
         var existingMt = await dbContext.ResourceContentVersionMachineTranslations
             .AsTracking()
-            .FirstOrDefaultAsync(x => x.Id == req.Id, ct);
-
-        if (existingMt is null || existingMt.UserId != user.Id)
+            .Include(rcvmt => rcvmt.ResourceContentVersion)
+            .ThenInclude(rcv => rcv.ResourceContent)
```

Comment:
Do we need these?

Reply:
I had thought we did so we can access the props of them, but I'll double check and make sure

Reply:
oh cool. I don't need them.